src/main/java/org/sagebionetworks/web/client/jsinterop/mui/Button.java

```diff
@@ -0,0 +1,66 @@
+package org.sagebionetworks.web.client.jsinterop.mui;
+
+import org.sagebionetworks.web.client.jsinterop.IconSvgProps;
+import org.sagebionetworks.web.client.jsinterop.React;
+import org.sagebionetworks.web.client.jsinterop.ReactComponentType;
+import org.sagebionetworks.web.client.jsinterop.SRC;
+
+public class Button
+  extends HasSx<ReactComponentType<ButtonProps>, ButtonProps> {
+
+  public Button() {
+    super(MaterialUI.Button, new ButtonProps());
+  }
+
+  public void setId(String id) {
+    props.id = id;
+    this.render();
+  }
+
+  public void setVariant(String variant) {
+    props.variant = variant;
+    this.render();
+  }
+
+  public void setColor(String color) {
+    props.color = color;
+    this.render();
+  }
+
+  public void setSize(String size) {
+    props.size = size;
+    this.render();
+  }
+
+  public void setStartIcon(String iconName) {
+    IconSvgProps iconProps = IconSvgProps.create(iconName, null);
+    props.startIcon =
+      React.createElement(SRC.SynapseComponents.IconSvg, iconProps);
+    this.render();
+  }
```

Comment:
Oh, I didn't realize you created the IconSvg here. I want this class to match the original interface as much as possible, but I like that you added this, as I think it is a really helpful utility to keep. What do you think of this refinement?

```suggestion
  /**
   * Convenience wrapper that uses synapse-react-client's IconSvg
   */
  public void setStartIcon(String iconSvgName) {
    IconSvgProps iconProps = IconSvgProps.create(iconSvgName, null);
    ReactNode icon = React.createElement(SRC.SynapseComponents.IconSvg, iconProps);
    this.setStartIcon(icon);
  }
  
  public void setStartIcon(ReactNode icon) {
    props.startIcon = icon
    this.render();
  }
```

Reply:
Makes sense! Though I don't see a ReactNode type.. can we use ReactElement?

Reply:
My bad, `ReactElement` will work! `ReactNode` is only in TypeScript and wouldn't make sense to represent in Java